### PR TITLE
VACMS-8291 kb login

### LINKED ISSUE #8289: Create way to login from KB
## User story
"as a user who is exploring the knowledge base but isn't logged in, I want to be able to navigate to login.

Related workflow:
- Start on login page > Click to KB page about login > then what?

ACs
- [ ] User can log in from any unauth KB page

### LINKED ISSUE #8291: Make KB the only CTA on login page
ACs
- [ ] Remove "Contact CMS help desk" link
- [ ] Revise link text from `Log in help` to `Get help`
- [ ] Should be the same on PIV and dev login options, as well as lower enviorns

Bonus: Add login block back to KB #8289 
(can wait until sprint 54 if necessary)

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
Closes #8291. Closes #8289. 

- We got overzealous on the new login page, Dave requested the one link with updated verbiage to direct users into the KB first before contacting help desk.
- Re-placed & re-exported the login block... appeared to be suffering from the same UUID issue that the cms announcement block did.
- Fixed styles for the login form that appears to anonymous users on KB pages
- Fixed the placeholder color for the search form in the header on KB pages. no ticket, this was pointed out by Dave during the CMSx backlog refinement today.

## Testing done
as a logged out user:
- verified there was only one link on the main log in page.
- Verified KB page had login available after following the link.
- Verified toggle works on KB login. 
- Verified the placeholder in the header KB search form passes contrast

## Screenshots
Screen Shot 2022-03-08 at 4 12 59 PM
Screen Shot 2022-03-08 at 4 13 05 PM
Screen Shot 2022-03-08 at 4 13 16 PM

## QA steps

- visit the login page, 
-  [x] see that the "get help" link is present on both PIV and user/pass login flows.
- [x] Click the "get help" link, see that the login form is present in the sidebar on KB pages
- [x] See that the search bar in the KB header has a placeholder that passes a11y contrast.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by tjheffner** [comment]
Not sure why phpstan failed but the behavioral test is a legit failure, will fix both in the morning.

**Comment by tjheffner** [comment]
Also appears whatever commits were brought in with this dependabot update are effecting some files i didn't touch at all... maybe they disappear when branch is updated with main. Will try after tests pass :shrug:

Screen Shot 2022-03-09 at 8 21 22 AM

File: config/sync/block.block.userlogin.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-uuid: 2f6fa543-f808-4a8c-b1a4-89d10a2f3c18
+uuid: d5702132-8565-4dee-bb03-3c8b130a2ade
```
Comment (swirtSJW) [inline config/sync/block.block.userlogin.yml:1]: Did you create a new block?  Seems weird that the uuid is changing.

edit: nvmind  I see that it is a new block.

**Review by swirtSJW — APPROVED** [review]
Nice work.  Checks out good.   
image

**Pull request merged**